### PR TITLE
vote15: Penalty Shoot-out

### PR DESCRIPTION
**Pull request by babaev1**
For the virtual rule book, we clarified the consequences of committing certain fouls during a penalty shoot-out. The same procedure could be applied to the physical rule book.

### CODE REVIEW

File: Section_II/competition_and_trophies.tex [hunk 1]
```diff
@@ -346,7 +346,34 @@
...
+
+The referee decides randomly towards which goal the penalty shoot-outs will be
+performed.
+Then, each penalty trial is performed with the following procedure.
+ 
+\begin{enumerate}
+\item The striker is defined as the robot from the striker's team with the
+  lowest ID without a red card.
+\item The goalkeeper is chosen based on the GameController information.
+\item Both players are spawned at appropriate locations.
```
Comment (MaikePaetzel) [inline Section_II/competition_and_trophies.tex:360]: Both players are placed or spawned at appropriate locations.

File: Section_II/competition_and_trophies.tex [hunk 2]
```diff
@@ -346,7 +346,34 @@
...
+\item State is set to SET
+\item Ball is spawned on the penalty mark.
```
Comment (MaikePaetzel) [inline Section_II/competition_and_trophies.tex:362]: The ball is placed or spawned...

File: Section_II/competition_and_trophies.tex [hunk 3]
```diff
@@ -346,7 +346,34 @@
...
+\item The referee waits 15 seconds before sending signals to start the trial
```
Comment (MaikePaetzel) [inline Section_II/competition_and_trophies.tex:363]: The referee waits 15 seconds before starting the trial

**Review by MaikePaetzel — CHANGES_REQUESTED** [review]

**Comment by babaev1** [comment]
Resolved

  
>Среда, 2 февраля 2022, 16:24 +03:00 от MaikePaetzel ***@***.***>:
> 
> 
***@***.*** requested changes on this pull request.
>----------------------------------------------------------------------
>In  Section_II/competition_and_trophies.tex :
>> @@ -346,7 +346,34 @@
> 
> 
> \bigskip
>+\added{
>+{\bfseries Penalty shoot-outs procedure}
>+
>+The referee decides randomly towards which goal the penalty shoot-outs will be
>+performed.
>+Then, each penalty trial is performed with the following procedure.
>+ 
>+\begin{enumerate}
>+\item The striker is defined as the robot from the striker's team with the
>+  lowest ID without a red card.
>+\item The goalkeeper is chosen based on the GameController information.
>+\item Both players are spawned at appropriate locations.
>
>Both players are placed or spawned at appropriate locations.
>----------------------------------------------------------------------
>In  Section_II/competition_and_trophies.tex :
>> @@ -346,7 +346,34 @@
> 
> 
> \bigskip
>+\added{
>+{\bfseries Penalty shoot-outs procedure}
>+
>+The referee decides randomly towards which goal the penalty shoot-outs will be
>+performed.
>+Then, each penalty trial is performed with the following procedure.
>+ 
>+\begin{enumerate}
>+\item The striker is defined as the robot from the striker's team with the
>+  lowest ID without a red card.
>+\item The goalkeeper is chosen based on the GameController information.
>+\item Both players are spawned at appropriate locations.
>+\item State is set to SET
>+\item Ball is spawned on the penalty mark.
>
>The ball is placed or spawned...
>----------------------------------------------------------------------
>In  Section_II/competition_and_trophies.tex :
>> @@ -346,7 +346,34 @@
> 
> 
> \bigskip
>+\added{
>+{\bfseries Penalty shoot-outs procedure}
>+
>+The referee decides randomly towards which goal the penalty shoot-outs will be
>+performed.
>+Then, each penalty trial is performed with the following procedure.
>+ 
>+\begin{enumerate}
>+\item The striker is defined as the robot from the striker's team with the
>+  lowest ID without a red card.
>+\item The goalkeeper is chosen based on the GameController information.
>+\item Both players are spawned at appropriate locations.
>+\item State is set to SET
>+\item Ball is spawned on the penalty mark.
>+\item The referee waits 15 seconds before sending signals to start the trial
>
>The referee waits 15 seconds before starting the trial
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>Triage notifications on the go with GitHub Mobile for  iOS or  Android .
>You are receiving this because you authored the thread. Message ID:  <RoboCup-Humanoid-TC/Rules/pull/26/review/870532901 @ github . com> 
 
 
--
A B
Отправлено из Почты  Mail.ru

**Comment by MaikePaetzel** [comment]
I don't see the commit that resolves these issues?

**Comment by MaikePaetzel** [comment]
According to the rule vote on G01Q15, this received a majority of the votes and this PR will hence be merged.

**Pull request merged**